### ISSUE TITLE
SMPJOB01 SMP failes with RC0008

### ISSUE THREAD

**Issue by moshix** [MVS-sysgen/sysgen#7]
This is from the master:

Job Name    Step Name    Proc Step Name    Completion Code
--------    ---------    --------------    ---------------
SMPJOB01    SMP          DLBUCL            0008 <--
 
  1 steps executed in selected jobs
  1 steps received completion code = 0008

        ***************
        *** ABORTED ***
        ***************
        
[!] Error encoutered exiting.

        ***************
        *** ABORTED ***
        ***************
        
[!] Error encoutered exiting.

**Comment by mainframed** [comment]
For unknown reasons this happens from time to time at various steps, if that happens you can restart that step with `./sysgen.sh --skip-distrib` and it should complete fine. If it fails at the exact same spot again and again then let us know and include the files `hercules.log` and `prt00e.txt` so we can see whats causing the issue. Just dealt with a similar issue on discord where the resolution was the same (rerun the step).

**Comment by wrljet** [comment]
As you know, I continue experiment with this process, and have learned more ways things don't work on various systems.
But I have not yet found any combination of Linux distro, compiler version, compilation flags, delays here and there,
or anything else that definitely "always works".   And some combinations are very bad.

I got it to run 6 or 7 times in a row recently on a Pi 4 with Ubuntu 21.04, but it finally crashed.
Actually crashed Hercules.  (93 minutes, BTW on that system)

As far as I'm concerned, there is a nasty bug lurking somewhere in Hercules.
And it's not going to be easy to find.

Bill

**Comment by moshix** [comment]
I guess what I am saying is that for me it crashes reliably. Ubuntu 16.04-latest

**Comment by wrljet** [comment]
I will give that a try this afternoon myself.

Please post what version of gcc you are using.  From `gcc --version`

Thanks,
Bill

**Comment by moshix** [comment]
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-linux-gnu/5/lto-wrapper
Target: x86_64-linux-gnu
Configured with: ../src/configure -v --with-pkgversion='Ubuntu 5.4.0-6ubuntu1~16.04.12' --with-bugurl=file:///usr/share/doc/gcc-5/README.Bugs --enable-languages=c,ada,c++,java,go,d,fortran,objc,obj-c++ --prefix=/usr --program-suffix=-5 --enable-shared --enable-linker-build-id --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --libdir=/usr/lib --enable-nls --with-sysroot=/ --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --with-default-libstdcxx-abi=new --enable-gnu-unique-object --disable-vtable-verify --enable-libmpx --enable-plugin --with-system-zlib --disable-browser-plugin --enable-java-awt=gtk --enable-gtk-cairo --with-java-home=/usr/lib/jvm/java-1.5.0-gcj-5-amd64/jre --enable-java-home --with-jvm-root-dir=/usr/lib/jvm/java-1.5.0-gcj-5-amd64 --with-jvm-jar-dir=/usr/lib/jvm-exports/java-1.5.0-gcj-5-amd64 --with-arch-directory=amd64 --with-ecj-jar=/usr/share/java/eclipse-ecj.jar --enable-objc-gc --enable-multiarch --disable-werror --with-arch-32=i686 --with-abi=m64 --with-multilib-list=m32,m64,mx32 --enable-multilib --with-tune=generic --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
Thread model: posix
gcc version 5.4.0 20160609 (Ubuntu 5.4.0-6ubuntu1~16.04.12)

**Comment by wrljet** [comment]
Moshix,

How many CPU cores, and how much RAM are you using?

I created a fresh Ubuntu 16.04.7 in VMware Workstation (2 CPUs, 4GB RAM), and gave this a test drive.

I first ran my modified version of the MVS-sysgen project.
(because it is working more reliably for me on other test systems)

I ran into two issues related to this older version of Ubuntu.
- `ncat` was not in the package manager.  So I built that from source.
- `SOFTWARE/BREXX/update.py` doesn't work with the older version of Python (3.5) on this system.
So I built a newer Python from source as well.

After that, the sysgen ran through completely!

I then ran Philip's current official sysgen.  It failed after about 15 minutes. 
Close to where yours failed, but not the same.  A bit further down.

I then reran it and it worked all the way through.  I repeated it, and got four runs successfully.

So that is the current status of my experimenting with Ubuntu 16.04.
More to come...

Bill
